YChartsLib/src/main/java/co/yml/charts/common/model/Point.kt

```diff
@@ -11,6 +11,8 @@ package co.yml.charts.common.model
 data class Point(
     val x: Float,
     val y: Float,
+    val xLable: String? = "",
```

Comment:
@KavinSethu Do we need extra label property here? Since we have an explicit method for overriding the same behavior